pydoctor/templates/apidocs.css

```diff
@@ -157,8 +158,14 @@ tr.private {
 }
 
 .fieldArg {
-    font-weight: bold;
-    color: #c7254e;
+    color: #222222;
+}
+
+/* Add an arrow before fields */ 
```

Comment:
I prefer to handle the field layout in a separate PR, since we'll be looking at moving the type information to a different place as well.